test/guest/fedora-atomic.setup

```diff
@@ -18,12 +18,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+# Use mac's in name policy, prevents interference from network-scripts
```

Comment:
I propose this change to check-networking instead:

```
diff --git a/test/check-networking b/test/check-networking
index b34b06c..4771879 100755
--- a/test/check-networking
+++ b/test/check-networking
@@ -117,6 +117,10 @@ class TestNetworking(MachineCase):
         m.execute("nmcli c reload")
         iface = self.add_iface(mac)
 
+        # Activate our settings on the new interface
+        wait(lambda: m.execute('nmcli d | grep %s | grep -v unavailable' % iface))
+        m.execute('nmcli c up "System %s"' % label)
+
         self.wait_for_iface(iface)
         b.click("#networking-interfaces tr[data-interface='%s']" % iface)
         b.wait_visible("#network-interface")
```

This makes the expected state more explicit.